client.go

```diff
@@ -227,8 +237,18 @@ func (c *Client) Post(url string, contentType ContentType, body interface{}, des
 			}
 			return nil
 		}
+	} else if resp.StatusCode >= 400 && resp.StatusCode < 500 {
```

Comment:
I'd prob account for 5xx too

Reply:
Can we make a card to address later? I'd like to get the dinghy fix out sooner then later, then return to make plank more consistent

Reply:
I think some services incorrectly return 500s, so it's prob important now

Reply:
Now as in, fix before merging this PR, or "we should fix this sooner than later"? I'm totally with you if it's the latter!

Reply:
your call. If we just shove the rest of they payload in the error message, then it's simple.